code/frontend/public/submitform.html

```diff
@@ -1,73 +1,182 @@
-              <option value="test" label="test data" id="test_data">test data</option>
+        <div class="dropdownfield">
+            <label for="Genre">Genres: </label>
+            <select name="genres" id="genre_select">
+              <option value="" disabled selected>--Select a Genre--</option>
+              <option value="test" id="test_data">test data</option>
             </select>
         </div>
-        <div>
+        <div class="dropdownfield">
             <label>Age Rating: </label>
             <select name="ratings" id="rating_select">
-              <option value="">--Please select an Age Rating--</option>
+              <option value="" disabled selected>--Select an Age Rating--</option>
               <option value="test">test data</option>
             </select>
         </div>
-        <div>
+        <div class="input">
             <label for="budget">Budget: </label>
-            <input type="number" id="budget" min="1" max="200000000" required>
+            <input placeholder="Enter a Budget"  type="number" id="budget" min="1" max="200000000" required>
         </div>
-        <div>
+        <div class="input">
             <label for="box_office">Box Office: </label>
-            <input type="number" id="box_office" min="1" max="1000000000" required>
+            <input placeholder="Enter a Box Office" type="number" id="box_office" min="1" max="1000000000" required>
         </div>
-        <div>
+        <div class="dropdownfield">
             <label>Cast: </label>
             <select name="casts" id="cast_select">
-              <option value="">--Please select a Cast Member--</option>
+              <option value="" disabled selected>--Select a Cast Member--</option>
               <option value="test">test data</option>
             </select>
         </div>
-        <div>
-            <label>Directors: </label>
+        <div class="dropdownfield">
+            <label for="Dicrector">Directors: </label>
             <select name="directors" id="director_select">
-              <option value="">--Please select a Director--</option>
+              <option value="" disabled selected>--Select a Director--</option>
               <option value="test">test data</option>
             </select>
         </div>
-        <div>
-            <label>Writers: </label>
+        <div class="dropdownfield">
+            <label for="Writer">Writers: </label>
             <select name="writers" id="writer_select">
-              <option value="">--Please select a Writer--</option>
+              <option value="" disabled selected>--Select a Writer--</option>
               <option value="test">test data</option>
             </select>
         </div>
-        <button type="submit">Generate</button>
+        <br>
+        <div class="buttons">
+
+            <div class="action_btn">
+
+                <button name="submit" class="action_btn submit" type="submit" value="Submit" onclick="myFunction()">Submit</button>
```

Comment:
try to refactor some of these styles to css

Reply:
i'm not sure what all in here is style, James worked on that portion. the only thing i think that attribution to the style are those class tags which are linking to style under those class names in the css

Reply:
You are correct, apologies